integration/cypress/support/index.ts

```diff
@@ -1,12 +1,22 @@
 /// <reference types="cypress" />
 
+import type { Options } from "cypress-axe";
 import "cypress-axe";
+import type { ContextObject } from "axe-core";
```

Comment:
Neither `Options` nor `ContextObject` are used in this file so should be removed.